### PR TITLE
use travis instead of jenkins

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

**Comment by knorth55** [comment]
test passing on Travis.

File: .travis.yml [hunk 1]
```diff
@@ -22,24 +24,24 @@ env:
...
-    - ROS_DISTRO=kinetic  USE_DEB=true
-    - ROS_DISTRO=kinetic  USE_DEB=false EXTRA_DEB="ros-kinetic-convex-decomposition ros-kinetic-ivcon"
+    - ROS_DISTRO=hydro   USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=hydro   USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-hydro-convex-decomposition ros-hydro-ivcon" BUILD_PKGS="jsk_pr2_calibration jsk_robot_startup pr2_base_trajectory_action jsk_pr2_startup jsk_pr2_desktop"
+    - ROS_DISTRO=indigo  USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=indigo  USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-indigo-convex-decomposition ros-indigo-ivcon"
+    - ROS_DISTRO=jade    USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=jade    USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-jade-convex-decomposition ros-jade-ivcon"
+    - ROS_DISTRO=kinetic USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=kinetic USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-kinetic-convex-decomposition ros-kinetic-ivcon"
```
Comment (k-okada) [inline .travis.yml]: you can add `USE_TRAVIES` at `global` section

Comment (knorth55) [reply .travis.yml]: thank you, i updated

**Comment by knorth55** [comment]
Updated: test passing on Travis.

**Pull request merged**